www/js/theme_editor.js

```diff
@@ -1,36 +1,26 @@
 const h = require('hyperscript');
-// const fs = require('fs');
 const Noty = require('noty');
+const fs = require('fs');
+const util = require('util');
+const imagemin = require('imagemin');
+const imageminJpegtran = require('imagemin-jpegtran');
+const imageminPngquant = require('imagemin-pngquant');
+const { remote } = require('electron');
+
 const themes = require('./themes.json');
 
-const { store, analytics } = require('electron').remote.require('./app');
+const mkdir = util.promisify(fs.mkdir);
+const userDataPath = remote.app.getPath('userData');
 
-// const imagesPath = 'assets/custom_backgrounds';
+const { store, analytics } = remote.require('./app');
 
 const defaultTheme = themes[0];
 
-/* defaultTheme.bgImage = '';
-const getCurrentTheme = () => {
-  const currentThemeString = localStorage.getItem('customTheme');
-  if (currentThemeString) {
-    try {
-      return JSON.parse(currentThemeString);
-    } catch (error) {
-      new Noty({
-        type: 'error',
-        text: `There is an error reading current theme.
-        Try checking theme file for errors. If error persists,
-        report it at www.sttm.co`,
-        timeout: 3000,
-        modal: true,
-      }).show();
-      return defaultTheme;
-    }
-  }
-  return defaultTheme;
-}; */
+const themesWithCustomBg = themes
+                            .filter(theme => theme.type === 'COLOR' || theme.type === 'SPECIAL')
+                            .map(theme => theme.key);
```

Comment:
This is not properly used yet, but will be used (and was inteded to be used) when supporting more than one theme.